gradio/components.py

```diff
@@ -75,8 +56,10 @@ def __init__(
         *,
         label: Optional[str] = None,
         show_label: bool = True,
-        requires_permissions: bool = False,
         interactive: Optional[bool] = None,
+        css: Optional[Dict] = None,
+        visible: bool = True,
```

Comment:
Is this needed? Since the parent `Block` class already defines a `visible` parameter int he constructor

Reply:
Yes, because we want type hints for everything.